src/main/java/dev/emi/trinkets/api/client/TrinketRendererRegistry.java

```diff
@@ -0,0 +1,19 @@
+package dev.emi.trinkets.api.client;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import net.minecraft.item.Item;
+
+public class TrinketRendererRegistry {
```

Comment:
Is there a reason to have this as a separate file from `TrinketsApi` other than separating server/client? I'm fine with this but I just want to hear your thought process.

Reply:
It's mainly just to separate server/client logic. More particularly, just making sure the client-only classes remain separate from everything else to avoid class-loading issues on servers. Originally I even had this named as `TrinketsClientApi` for putting any client-only logic, but I wasn't really sure if we were going to have anything that falls under this category other than this rendering code.

Reply:
We can rename it if we need to sounds good for now.